src/Subscription/FixedTime.php

```diff
@@ -45,4 +48,28 @@ public function getPeriod(): \DateInterval
     public function assertValid(Core\Activity $activity): void
     {
     }
+
+    /**
+     * @param int $id
+     */
+    public function persists(int $id): void
+    {
+        $this->id = $id;
+    }
+
+    /**
+     * @return \stdClass
+     */
+    public function jsonSerialize(): \stdClass
+    {
+        $object = new \stdClass();
+        $object->id = $this->id;
+        $object->type = get_class($this);
+        $object->startDay = $this->dateTime;
+        $object->period = date_create('@0')->add($this->period)->getTimestamp();
+        $object->activity = null;
```

Comment:
Property `activity` makes no sense for this subscription. No need to define it.

Reply:
Fixed